modules/recording/JibriQueue.js

```diff
@@ -0,0 +1,233 @@
+/* global $ */
+
+import EventEmitter from 'events';
+import { getLogger } from 'jitsi-meet-logger';
+import { $iq, Strophe } from 'strophe.js';
+
+const logger = getLogger(__filename);
+
+/**
+ * A static counter of the JibriQueue instances used to generate a unique ID for the JibriQueue instances.
+ */
+let id = 0;
+
+/**
+ * Represents a jibri queue.
+ */
+export default class JibriQueue extends EventEmitter {
```

Comment:
how about extending `Listenable` which is kind of LJM layer for event emitter instead of extending library import directly?